pkg/linksharing/sharing/assets/templates/single-object.html

```diff
@@ -26,38 +69,29 @@
           <img class="embed-responsive embed-responsive-4by3" id="imgTag" alt="preview image">
           <video class="embed-responsive embed-responsive-4by3" id="videoTag" controls></video>
           <audio class="embed-responsive embed-responsive-4by3" id="audioTag" controls></audio>
-          <img class="embed-responsive embed-responsive-4by3 d-none" id="placeholderImage" src="{{.Base}}/static/img/file-drop.svg?v={{.VersionHash}}" alt="placeholder image">
-        </div>
-      </div>
-
-    </div>
-
-    <div class="col-lg-4 col-xl-3 px-lg-4 px-xl-5 sidebar active d-block">
-      <div class="row mb-3 mt-3">
-        <div class="col text-center text-md-left">
-          <a href="https://www.storj.io/"><img src="{{.Base}}/static/img/logo.svg?v={{.VersionHash}}" class="logo mt-4" alt="Storj Logo"></a>
-        </div>
-      </div>
-      <div class="row">
-        <div class="col">
-          <div class="row justify-content-center">
-            <div class="col py-3 py-lg-4 text-center text-md-left">
-              <h3 class="mb-3">You’re getting this file from all over the world.</h3>
-              <p>Storj splits files into smaller pieces, then distributes those pieces over a global network of nodes and recompiles them securely on download. This map shows the real-time locations of this file’s pieces.</p>
-              <p class="nodes-count w-100 mt-2 text-center">{{ .Data.NodesCount }} Storj nodes are storing a piece of this file</p>
+          <!-- <img class="embed-responsive embed-responsive-4by3 d-none" id="placeholderImage" src="{{.Base}}/static/img/file-drop-v4.png?v={{.VersionHash}}" alt="placeholder image"> -->
```

Comment:
@boshevski why are you keeping this element commented out?

Reply:
Hey Artur, thanks for spotting this. I was designing and testing different placeholder images, and at the end I replaced it with a CSS only visual. This will save on bandwidth from all the linksharing files that don't have a preview. I forgot about this, so feel free to remove it.

Reply:
Here is a fix for it: https://github.com/storj/edge/pull/473